SU2_CFD/src/output/COutput.cpp

```diff
@@ -476,7 +479,7 @@ void COutput::WriteToFile(CConfig *config, CGeometry *geometry, OUTPUT_TYPE form
       extension = CSU2MeshFileWriter::fileExt;
 
       if (fileName.empty())
-        fileName = volumeFilename;
+        fileName = config->GetFilename(volumeFilename, "", curTimeIter);
```

Comment:
So if the filename is empty, the time iteration is added, but if the filename is not empty,  the iterations were always already added before to `fileName`, is that it?

Reply:
Essentially I have followed all the other cases. Without appending the time iter at the end of the file name it just overwrites the same file at each unsteady iteration.